src/modules/feedback/i18n/feedback.provider.js

```diff
@@ -2,57 +2,77 @@ export const provide = (lang) => {
 	switch (lang) {
 		case 'en':
 			return {
-				feedback_mapFeedback_header: 'Send Feedback',
-				feedback_mapFeedback_could_not_save: 'Your feedback could not be saved',
-				feedback_mapFeedback_saved_successfully: 'Your feedback was successfully submitted',
-				feedback_mapFeedback_categorySelection: 'Category',
+				feedback_generalFeedback_changeDescription_error: 'Required field',
+				feedback_generalFeedback_changeDescription_helper: 'Required field',
```

Comment:
Let's replace all individual error keys with one valid for all feedback components,
e.g:
```
feedback_required_field: 'Required field'
feedback_invalid_eMail: 'No valid e-mail address
```